modules/tinymce/src/core/main/ts/annotate/Identification.ts

```diff
@@ -42,19 +45,31 @@ const identify = (editor: Editor, annotationName: Optional<string>): Optional<{u
 
 const isAnnotation = (elem: any): boolean => SugarNode.isElement(elem) && Class.has(elem, Markings.annotation());
 
+const offscreenSelectionClass = 'mce-offscreen-selection';
```

Comment:
This might be better at the top, as constants normally are at the top of the file instead of hidden in the middle somewhere?